devices/common_patches/sai/sai_v1.yaml

```diff
@@ -9,3 +9,33 @@
       "?SR": {}
       "?CLRFR": {}
       "?DR": {}
+  "?CR1":
+    _modify:
+      OutDri:
+        name: OUTDRIV
+      SAI?EN:
+        name: SAIEN
```

Comment:
I get why you do this and I think it's the best compromise, but jeez it sure sucks that they have SAIAEN and SAIBEN in otherwise identical register blocks.